### PR TITLE
HR banks : initial commit

### PR DESCRIPTION
**Pull request by badbole**
Initial module containig banks data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ Odoo Croatian Localization
...
+[//]: # (addons)
+
+Available addons
+----------------
+addon | version | summary
+--- | --- | ---
+[l10n_hr_bank](l10n_hr_bank/) | 10.0.0.0.1 | Import all Croatia banks with BIC, for personal accounts added types: tekuci, ziro, zasticeni
+
+[//]: # (end addons)
+
```
Comment (dreispt) [inline README.md]: Don't touch this README file: the module list will be automatically generated by an OCA Bot

File: l10n_hr_bank/README.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+=================
+
+* This module will import current list of banks in Croatia
+* Partner bank model extend with field account purpose,
+  selection is mandatory only for personal bank accounts opened in croatian banks,
+  in evidention of croatian companies
+* This module is not dependant on module base_bank_prefix,
+  but bank data contains prefix info, so, if you decide to install and use
+  base_bank_prefix after installing this module,
+  you will need to update this module for prefixes to be fully loaded into database
```
Comment (dreispt) [inline l10n_hr_bank/README.rst]: Banks are not HR specific, and are useful/needed in other areas (such as finance).
Also, module prefix should include the country initials ("cr"?).
Consider naming the module `l10n_cr_bank`

**Comment by dreispt** [comment]
This repo is missing the `.travisci.yml`to run the automated checks.
For an example, you can look at https://github.com/OCA/l10n-finland

File: l10n_hr_bank/README.rst (hunk 2 above)
Comment (badbole) [reply l10n_hr_bank/README.rst]: @dreispt good point with "do not touch readme file" - squashed commit without readme,
as for HR prefix, usualy this stands for Human resources, but, Croatia country code is HR ,
(actualy  l10n_cro would be acceptable) but, 
we have l10n_hr as standard localization for Croatia in odoo addons, all croatian translations are 'hr.po' or 'hr_HR.po' so naming module l10n_cr_* would be contradictory to standards and rules for localizations. I belive lots of modules in this repo will be prefixed l10n_hr, 
so l10n_hr_bank should be valid module name. ( and l10n_hr_hr for localization of HR segment will also be acceptable?) 
I will add travis.yml,

Comment (badbole) [reply l10n_hr_bank/README.rst]: @dreispt update: indeed there is not .travisci.yml as in finland localization that you mentioned, 
instead, there is .travis.yml , as in all other localisations (agrentina, belgium, france, netherlands...)
and all checks are started normaly

File: l10n_hr_bank/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline l10n_hr_bank/__init__.py]: "coding" instead of "encoding"

File: l10n_hr_bank/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Croatia - Banking",
+    "summary": "Croatia Banking localization",
+    "category": "Localisation / Croatia",
+    "images": [],
+    "version": "10.0.0.0.1",
+    "application": False,
+    "author": "Odoo Community Association (OCA)",
```
Comment (dreispt) [inline l10n_hr_bank/__manifest__.py]: Add your name or company here.

File: l10n_hr_bank/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline l10n_hr_bank/models/__init__.py]: coding

**Review by dreispt — COMMENTED** [review]
LGTM - only a few minor comments to fix.

**Review by dreispt — APPROVED** [review]

**Comment by badbole** [comment]
@dreispt  -  thank your for reviwing this, 
hope i can call you to review few next modules in this localisation...
all done and squashed into initial commit

**Comment by dreispt** [comment]
You're welcome.
I'll try to have a look.

**Pull request merged**